engines/ags/detection_tables.h

```diff
@@ -1920,6 +1923,9 @@ const PlainGameDescriptor GAME_NAMES[] = {
 	{ "osdscourge", "Oceanspirit Dennis: Scourge of the Underworld" },
 	{ "osdscourgehd", "Oceanspirit Dennis: Scourge of the Underworld HD" },
 	{ "osdsearch", "The Search for Oceanspirit Dennis" },
+	{ "osdshindig", "The Shindig of Dennis" },
+	{ "osdsquares", "Oceanspirit Dennis: [][][][][][][][][]" },
```

Comment:
Are these part of the actual game title?

Reply:
Yes it's weird but that's the name on the official website
http://www.barnrunner.com/OSD/ (see number #15)
The squares are unicode 25AF, but that character doesn't show on scummvm gamelist, so I used the square brackets.

Reply:
Thanks! Odd naming...